pkg/client/ghcr/ghcr.go

```diff
@@ -40,21 +40,26 @@ func (c *Client) Tags(ctx context.Context, host, owner, repo string) ([]api.Imag
 	tc := oauth2.NewClient(ctx, ts)
 	client := github.NewClient(tc)
 
-	opt := &github.ListOptions{PerPage: 10}
-	releases, _, err := client.Repositories.ListReleases(ctx, owner, repo, opt)
+	var tags []api.ImageTag
+	opt := &github.ListOptions{PerPage: 50}
+	for {
+		releases, resp, err := client.Repositories.ListReleases(ctx, owner, repo, opt)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get github Releases: %s", err)
+		}
 
-	if err != nil {
-		return nil, fmt.Errorf("failed to get github Releases: %s", err)
-	}
+		for _, rel := range releases {
+			tags = append(tags, api.ImageTag{
+				Tag:       *rel.TagName,
```

Comment:
In practice I suspect its never the case but you could still check if this value is nil.
```
		for _, rel := range releases {
                        if rel.TagName == nil {
                            continue
                        }
			tags = append(tags, api.ImageTag{
				Tag:       *rel.TagName,
				SHA:       "",
				Timestamp: rel.PublishedAt.Time,
			})
		}
```